### PR TITLE
remove unused use statements

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by jails** [comment]
Hi, `spec/fixture` only contains examples of PHP files. So they don't intend to be meaningful, they are just file example used to check that JIT processing is done right. So the `use` statements can't be removed from `spec/fixture`.

File: src/cli/Kahlan.php [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
 use jit\Interceptor;
 use jit\Patchers;
 use filter\Filter;
 use filter\behavior\Filterable;
 use kahlan\Suite;
-use kahlan\Matcher;
-use kahlan\cli\Cli;
-use kahlan\cli\Args;
-use kahlan\jit\patcher\DummyClass;
+use kahlan\Matcher;use kahlan\jit\patcher\DummyClass;
```
Comment (jails) [inline src/cli/Kahlan.php]: Glitch ?

Comment (samsonasik) [inline src/cli/Kahlan.php]: updated ;)

**Comment by samsonasik** [comment]
@jails done, I've rolled back `spec/fixture` change ;)

**Comment by jails** [comment]
Cool :+1: !
Would you mind squashing commits all together to keep clean commit history ?

**Comment by samsonasik** [comment]
done ;), squashed ;)

**Comment by jails** [comment]
Thank you !

**Pull request merged**